### PR TITLE
[EOC-175] Bug handle missing avatar

### PR DESCRIPTION
**Pull request by adamklepacz**
Zrzut ekranu 2019-05-20 o 11 33 16

### CODE REVIEW

File: server/controllers/list.js [hunk 1]
```diff
@@ -777,14 +777,15 @@ const addViewer = (req, resp) => {
...
+      const userExists = checkIfArrayContainsUserId(viewersIds, newMemberId);
+
+      if (userExists) {
+        throw new BadRequestException('User is already a member.');
+      }
 
       user = userData;
 
-      if (userNotExists) {
+      if (!userExists) {
```
Comment (marekrozmus) [inline server/controllers/list.js]: this check is not needed - this code won't be run as exception will be thrown in line 783

File: client/src/common/components/Members/components/MemberButton.jsx [hunk 2]
```diff
@@ -1,22 +1,41 @@
...
-    <img
-      alt={`${member.displayName} avatar`}
-      className="member-button__avatar"
-      onClick={onDisplayDetails}
-      src={member.avatarUrl}
-      title={member.displayName}
-    />
-  </button>
-);
+import UserIconPlaceholder from 'assets/images/user.svg';
```
Comment (alemikolo) [inline client/src/common/components/Members/components/MemberButton.jsx]: I'm not sure, but if we have to import images we should use '../../assets/images/user.svg' now.

Comment (adamklepacz) [reply client/src/common/components/Members/components/MemberButton.jsx]: @marekrozmus   Should it be a full relative path?

**Comment by alemikolo** [comment]
Comment is one more place, that this solution should be implemented.

File: client/src/common/components/Comments/Comment.jsx [hunk 3]
```diff
@@ -1,37 +1,58 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import Linkify from 'react-linkify';
+import classNames from 'classnames';
 
 import { UserIcon } from 'assets/images/icons';
 import { dateFromString } from 'common/utils/helpers';
+import UserIconPlaceholder from '../../../assets/images/user.svg';
```
Comment (marekrozmus) [inline client/src/common/components/Comments/Comment.jsx]: why not just 'assets/images/user.svg'?

File: client/src/common/components/Members/components/MemberButton.jsx [hunk 4]
```diff
@@ -1,22 +1,41 @@
...
-    <img
-      alt={`${member.displayName} avatar`}
-      className="member-button__avatar"
-      onClick={onDisplayDetails}
-      src={member.avatarUrl}
-      title={member.displayName}
-    />
-  </button>
-);
+import UserIconPlaceholder from '../../../../assets/images/user.svg';
```
Comment (marekrozmus) [inline client/src/common/components/Members/components/MemberButton.jsx]: same here

File: client/src/common/components/Toolbar/components/UserBar/UserBar.jsx [hunk 5]
```diff
@@ -9,9 +9,11 @@ import { getCurrentUser } from 'modules/authorization/model/selectors';
 import { UserPropType } from 'common/constants/propTypes';
 import { LogoutIcon, UserIcon, CohortIcon } from 'assets/images/icons';
 import Overlay, { OverlayStyleType } from 'common/components/Overlay';
+import UserIconPlaceholder from '../../../../../assets/images/user.svg';
```
Comment (marekrozmus) [inline client/src/common/components/Toolbar/components/UserBar/UserBar.jsx]: and here

**Pull request merged**